packages/app-polkadot-ethereum-bridge/src/Bridge.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import Web3 from 'web3';
+
+import { Box } from "@material-ui/core";
+
+// Local imports
+import {TEST_TOKEN_CONTRACT_ADDRESS, APP_ETHEREUM_CONTRACT_ADDRESS, APP_ERC20_CONTRACT_ADDRESS } from "./config";
+import AppEthereum from "./AppEthereum";
```

Comment:
As per my previous comment about naming we should consider having this instead:
```suggestion
import AppETH from "./AppETH";
```